docs/conf/portland/2022/news/welcome.rst

```diff
@@ -0,0 +1,93 @@
+:template: {{year}}/generic.html
+
+.. post:: Dec 15, 2022
+   :tags: {{shortcode}}-{{year}}, website, tickets
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs Portland {{year}} <https://www.writethedocs.org/conf/portland/{{year}}/>`_.
+We're excited to announce the dates for our next conference:
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We are planning to do a **hybrid event** this year,
+which means the in-person part of the conference will be returning.
+We are still in the planning phases for how to handle this safely,
+and would love your `feedback <mailto:portland@writethedocs.org>`_ on how we should progress.
+
+.. TODO: Add link to survey?
+
+It's been a rough year for a lot of us,
+but we're hoping that late next spring things will be looking a lot safer for us to gather.
+We plan to release more details during the next few weeks,
+but there's a few specific updates below.
+
+Health Guidelines
+-----------------
+
+We will continue to monitor the health guidelines for Portland & Oregon as the conference approaches,
+but our current plan is to have the following rules in place:
+
+* Vaccination required for all attendees
+* Masks required for all indoor spaces
+* A new venue that provides more access to outside air, and outdoor space
+* 100% refund for all attendees who need to cancel for COVID-related reasons
+* Limited venue capacity to give people enough room to socially distance
+* Seating options based on comfort level being near others in various spaces
+* Venue spaces with windows that open or that are fully outside
+
+We also have a few more ideas to ensure safety of all attendees that we are discussing,
+and will have more information as the event comes closer.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We plan to launch our CFP in January and have it last until sometime in February.
+We're still working out the exact details for the CFP,
+but our current plan is to record all talks so that the experience will be comparable between online and in-person attendees.
+We hope this will reduce anxiety for our speakers about feeling the need to attend in person,
+or have a second-rate experience presenting remotely.
```

Comment:
```suggestion
We hope that this will help maintain the accessibility level that remote presentations gave us and invite speakers who cannot or do not feel comfortable attending in person.
```

Reply:
I'd rather focus on accessibility than anxiety ;)